frontend/packages/console-shared/src/components/virtualized-grid/Grid.scss

```diff
@@ -0,0 +1,6 @@
+.ocs-grid {
+  .ReactVirtualized__Grid__innerScrollContainer {
+    width: 100% !important;
+    max-width: 100% !important;
```

Comment:
Why is this needed? I removed this and it worked.

Reply:
This was added to give inner container full width, but it's not necessary anymore. Removed it.